sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BigQueryReadWriteIT.java

```diff
@@ -154,6 +156,78 @@ public void testSQLRead() {
     assertEquals(state, State.DONE);
   }
 
+  @Test
+  public void testSQLRead_withDirectRead() {
```

Comment:
We should probably also add a test that EXPORT works, so when that is no longer the default we will still have a test for it.

Reply:
Added an integration test for BigQuery read in EXPORT mode, fixed.